soc/rockchip/soc.yml

```diff
@@ -7,3 +7,6 @@ family:
       - name: rk3568
         socs:
           - name: rk3568
+      - name: rk3588s
+        socs:
+          - name: rk3588s
```

Comment:
is there really a line of SoCs as a series called `rk3588s` with a specific SoC in that line called `rk3588s`?

Reply:
@nordicjm 
I am bit unsure about it as its not under any series on Rockchip's [website](https://www.rock-chips.com/a/en/products/RK35_Series/index.html) therefore, I made a SoC series for it. However, it might fall under the rk35 series. Do I change it to that ?

Reply:
yes, change to:
```
      - name: rk35
        socs:
          - name: rk3568
          - name: rk3588s
```
and update the existing Kconfig for SOC_SERIES